File: lbrtrn2tsk2.py
rzh = 1998
rvnrzhnst = 0.04
cl = 30 
gd = 1998
while rzh < cl:
    gd += 2
    rzh += rzh * rvnrzhnst
print("Урожайность превысит 30 центнеров с га в", gd, "году")

File: lbrtrn2tsk2.pas
program lbrtrn2tsk2;
var gd:Integer; clrzh,rzh:Real;
begin
gd:= 1998;
rzh:= 20;
clrzh:= 30;  
while rzh <= clrzh do
begin
rzh:= rzh * 1.04;  
gd:=gd + 2;            
end;
WriteLn('Урожайность превысит ', clrzh, ' центнеров с га в ', gd, ' году.');
end.